test/pretty-print.test.js

```diff
@@ -0,0 +1,100 @@
+'use strict'
+
+const t = require('tap')
+const test = t.test
+const Fastify = require('..')
+
+test('pretty print - static routes', t => {
+  t.plan(2)
+
+  const fastify = Fastify()
+  fastify.get('/test', () => {})
```

Comment:
Could you add '/test/' here?

Reply:
I don't see why it can be useful, this test is just a replica of https://github.com/delvedor/find-my-way/blob/master/test/pretty-print.test.js, is just needed to check that the api is working correctly :)